score.py

```diff
@@ -145,20 +146,15 @@ def get_shiny_bonus(save: Savefile) -> Score:
     return Score("Shiny", score=score)
 
 def get_max_hp_bonus(save: Savefile) -> Score:
-    """Get points based on how much max HP gained"""
-    max_hp_tuple = namedtuple("MaxHP", ["base_max_hp", "asc_max_hp_loss"])
+    """Get points based on how much max HP gained, ignores Ascension 14 penalty."""
     max_hp_dict = {
-        "Ironclad": max_hp_tuple(80, 5),
-        "Silent": max_hp_tuple(70, 4),
-        "Defect": max_hp_tuple(75, 4),
-        "Watcher": max_hp_tuple(72, 4)
+        "Ironclad": 80,
+        "Silent": 70,
+        "Defect": 75,
+        "Watcher": 72
     }
 
-    character_starting_max_hp = max_hp_dict[save.character].base_max_hp
-    if save.ascension_level >= 14:
-        character_starting_max_hp -= max_hp_dict[save.character].asc_max_hp_loss
-
-    max_hp_gain = save.max_health - character_starting_max_hp
+    max_hp_gain = save.max_health - max_hp_dict[save.character]
```

Comment:
It is possible that `save.character` is modded and not in the dict

Reply:
Don't really think metric_max_hp_per_floor is super reliable if we get a Max HP bonus/penalty from Neow, but I suppose we can calculate the starting max hp using the NeowBonusLog. So maybe take metric_max_hp_per_floor[0] +- NeowBonusLog.maxHpGained or maxHpLost. Think this might be the best solution?

It might depend if metric_max_hp_per_floor starts on floor 1 or floor 0. We'd also need to take into account getting Strawberry or Mango from Neow.

OR we can just ignore this bonus if the character is modded and keep the logic but at least don't break if it's a modded char

Reply:
The list starts on floor 1; everything from Neow can only be gotten via RHP. And whichever way you want to go is fine by me; I'm fine with just not working but not breaking on a modded character.

Reply:
Ok I think I'll just make it not break. My reasoning for this is because Well fed and Stuffed ignore the Ascension 14 max HP loss, and there's no way for us to get the base Max health of a character if it's over A14.

ex: Silent will only get Well fed if `max HP >= 85` even if her max HP at the start of an A20 run is 66 and it seems like she should get Well fed if `max HP >= 81`

Reply:
Makes sense. I believe `save.neow_bonus.max_hp` has the data, but I also think it breaks on modded characters lol (something to fix at some point!)

Reply:
Ya seems like it has the exact same problem of not being able to truly know the modded character's Base max HP. I don't think that's a PR that belongs in RHP really, perhaps Basemod proper. No idea

Reply:
Yeah, such is life! I'm fine not supporting them. Modded runs always break things a little bit, so I'm fine with them being a bit broken on the run pages.